StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebViewController.swift

```diff
@@ -40,17 +43,19 @@ class ConnectComponentWebView: ConnectWebView {
     init<InitProps: Encodable>(
         componentManager: EmbeddedComponentManager,
         componentType: ComponentType,
+        loadContent: Bool,
```

Comment:
I made `loadContent` a required param to ensure we always explicitly pass the value from the Component manager (I wasn't doing this for AccountManagement, PaymentDetails, or NotificationBanner which meant it was loading in tests)